resources/qml/Preferences/ProfilesPage.qml

```diff
@@ -173,7 +175,7 @@ UM.ManagementPage
 
             Label {
                 id: defaultsMessage
-                visible: currentItem && !currentItem.metadata.has_settings
+                visible: false
```

Comment:
Because I have no proper solution for it. The check that is there currently is broken since when the quality_changes container is empty that does not mean the underlying quality container is empty. I found the entire thing rather dubious so I hid it until I could discuss it, but then forgot to discuss it.